tsconfig.json

```diff
@@ -31,7 +31,10 @@
     "exclude": [
         "dist",
         "projects/igniteui-angular/migrations/",
-        "projects/igniteui-angular/schematics/"
+        "projects/igniteui-angular/schematics/",
+        "projects/igniteui-angular/cypress/",
+        "cypress/",
```

Comment:
Oh so this is the main tsconfig? Assuming cypress run would have their own tsconfig this is fine, but also kind of weird. As far as I can see typedoc has options below and those are based on this exclude list, yet there are entry point options.

So the config can be improved in the future. We're interested in API for the project lib only

Reply:
Actually, there's a [`tsconfig.typedoc.json`](https://github.com/IgniteUI/igniteui-angular/blob/f9db7af5e3b6b7504ecbc6ca3cb0bfee0f94fd22/tsconfig.typedoc.json) already present in the repo. What's lacking is it doesn't seem to be in use for the actual command:
https://github.com/IgniteUI/igniteui-angular/blob/f9db7af5e3b6b7504ecbc6ca3cb0bfee0f94fd22/gulpfile.js#L234-L240
Replace the `tsconfig.json` on L240 w/ `tsconfig.typedoc.json` and it should work fine